src/13_file_io.py

```diff
@@ -9,10 +9,35 @@
 # Print all the contents of the file, then close the file
 
 # YOUR CODE HERE
+with open('foo.txt') as f:
+  print(f.read())
```

Comment:
Excellent work using `with open()`. This way, `f.close()` is called automatically!